### ISSUE TITLE
Migration 002_slug fails on postgres

### ISSUE THREAD

**Issue by marcuspp** [hovel/pybbm#188]
On the latest version 0.17.0 and on Django 1.8  using postgres I get the following error trying to run the django migrations. On migration 002_slug.
```
django.db.utils.OperationalError: cannot ALTER TABLE "pybb_forum" because it has pending trigger events
```

I think this is to do with the migration trying to update the schema and update data in the same transaction block., which postgres does not allow.

Please investigate and fix.

Thanks.

**Comment by GeyseR** [comment]
Hi, @marcuspp!
Thanks for reporting this issue. 
I've just released 0.17.1 version, can you try to apply migrations from this version?

**Comment by marcuspp** [comment]
hi @GeyseR,

Thanks for the prompt fix. I've installed the new version.

I now get the following error:
```
CommandError: Conflicting migrations detected (0002_slugs_optional, 0004_slugs_required, 0003_slugs_fill in pybb).
To fix them run 'python manage.py makemigrations --merge'
```

But if I try the merge them and run the migrations that fails with a django.core.exceptions.FieldDoesNotExist like you'd expect.

**Comment by marcuspp** [comment]
I managed to get it working by manually changing the dependencies so the are forced to run in order. At the moment they all just depend on 0001_initial

**Comment by GeyseR** [comment]
Thanks for posting results there. I accidentally forget to change migrations dependencies after copy-paste source migration.

**Comment by GeyseR** [comment]
released 0.17.2 to fix last problem. this should fix slug migrations for Postgres database
